packages/create-astro/src/actions/template.ts

```diff
@@ -24,7 +24,7 @@ export async function template(
 				{ value: 'basics', label: 'A basic, helpful starter project', hint: '(recommended)' },
 				{ value: 'blog', label: 'Use blog template' },
 				{ value: 'starlight', label: 'Use docs (Starlight) template' },
-				{ value: 'minimal', label: 'A minimal (empty) starter' },
+				{ value: 'minimal', label: 'Use minimal template with no extra files' },
```

Comment:
This seems less clear to me. "Extra" compared to what?

Reply:
We can revert to the original (empty) wording if it was better!